docs/toolkits/openstack-planning-guide.md

```diff
@@ -1,11 +1,124 @@
+
+It should not be under-stated how convenient it is to have someone else maintain a cloud for you! This toolkit will get you started with a basic cloud system, but to design, build, operate, and maintain a performant, large-scale, production-quality cloud, you must develop specialized skills on your team (or in yourself), enumerated in the section below. Any IT engineer can learn these skills, but they go well beyond basic Linux systems administration. At a research institution, these are difficult skills to hire for. This guide will get you on your way, but understand it is a journey.
+
+For smaller organizations, and organizations with less technology-demanding missions, savvy consumption of commercial cloud services may support the mission more efficiently than building and operating an OpenStack cloud in-house. Researchers at US-based organizations may also qualify for an allocation on [Jetstream2](https://jetstream-cloud.org), a national-scale, production-quality OpenStack cloud for science and engineering research.
+
+### What skills do I need to build a cloud?
+
+To build a cloud with this guide, you need basic Linux systems administrator skills. These are detailed in the [OpenStack Automated Installation Guide](openstack-automated-installation.md#prerequisite-skills).
+
+To build a large-scale highly-available cloud to support large, downtime-sensitive, and/or mission-critical workloads, you also need to develop familiarity with:
+
+- Distributed data storage systems such as [Ceph](https://en.wikipedia.org/wiki/Ceph_(software))
+- Software-defined networking technologies such as [VLAN](https://en.wikipedia.org/wiki/VLAN)s, [VXLAN](https://en.wikipedia.org/wiki/Virtual_Extensible_LAN), Open vSwitch, possibly [BGP](https://en.wikipedia.org/wiki/Border_Gateway_Protocol)
+- TODO, finish this
+
+### OpenStack Services and Components
+
+[![OpenStack Diagram](../img/openstack-diagram.jpeg)](../img/openstack-diagram.jpeg)
+
+The most fundamental OpenStack services to know about are:
+
+| Service type | OpenStack name | What it delivers to users               | AWS Equivalent |
+|--------------|----------------|-----------------------------------------|----------------|
+| identity     | Keystone       | Users and access control for the cloud  | IAM            |
+| disk image   | Glance         | Operating systems for virtual computers | N/A            |
+| computing    | Nova           | Virtual computers                       | EC2            |
+| networking   | Neutron        | Software-defined network resources      | VPC            |
+
+Some other commonly-adopted services include:
+
+| Service type            | OpenStack name | What it delivers to users               | AWS Equivalent     |
+|-------------------------|----------------|-----------------------------------------|--------------------|
+| dashboard               | Horizon        | A web interface for using OpenStack     | Management Console |
```

Comment:
Would it be too pork-barrel-esque to put Exosphere here?

Reply:
Nope. I'll just want to be clear that it's not an official OpenStack project